plugins/auth0/go.mod

```diff
@@ -1,55 +1,48 @@
 module github.com/aserto-dev/ds-load/plugins/auth0
 
-go 1.21
+go 1.23.2
 
 replace github.com/aserto-dev/ds-load/sdk => ../../sdk
 
+replace github.com/aserto-dev/ds-load/cli => ../../cli
```

Comment:
Why do plugins need to import the `cli` package? I don't see it used anywhere.

Reply:
moved the cc from cli to sdk